src/Components/NavBar/NavBarMobileMenu/NavBarMobileMenuNotifications.tsx

```diff
@@ -25,8 +25,11 @@ export const NavBarMobileMenuNotifications: React.FC<NavBarMobileMenuNotificatio
   const { trackEvent } = useTracking()
   const unreadConversationCount = me?.unreadConversationCount ?? 0
   const unreadNotificationsCount = me?.unreadNotificationsCount ?? 0
+  const unseenNotificationsCount = me?.unseenNotificationsCount ?? 0
   const hasConversations = unreadConversationCount > 0
   const hasNotifications = unreadNotificationsCount > 0
+  const hasUnseenNotifications = unseenNotificationsCount > 0
+  const displayBlueDot = hasNotifications && hasUnseenNotifications
```

Comment:
**suggestion**: rename `displayBlueDot` to `shouldDisplayIndicator`. what do you think?

Reply:
done!

Reply:
thanks!